### PR TITLE
Signup: pointing to the proper developer welcome page

### PR DESCRIPTION
**Pull request by rralian**
The signup/register work was done separate from the devdocs/welcome work, so we need to wire it up.

## Testing
@nb could you give this a runthrough? Test both register and login and make sure it takes you to the proper /devdocs/start page.

### CODE REVIEW

**Comment by nb** [comment]
Login works great.

File: client/signup/config/flows.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ const flows = {
 
 	developer: {
 		steps: [ 'themes', 'site', 'user' ],
-		destination: '/me/next?welcome',
+		destination: '/devdocs/welcome',
```
Comment (mtias) [inline client/signup/config/flows.js]: Should we take any precautions in environments that don't have devdocs enabled?

Comment (rralian) [inline client/signup/config/flows.js]: I checked with @scruffian and a signup flow only shows up when you link directly to the flow (i.e., via `/start/developer`) and that only gets linked to from `devdocs/controller.js`... which is only available in an environment with `devdocs` enabled. So I don't think any other checks are necessary at this point.

**Comment by nb** [comment]
Signup works great, too.

**Comment by nb** [comment]
Let’s do it :) :shipit:

**Pull request merged**